flow/connectors/postgres/postgres.go

```diff
@@ -620,18 +618,11 @@ func (c *PostgresConnector) NormalizeRecords(ctx context.Context, req *model.Nor
 	for _, destinationTableName := range destinationTableNames {
 		normalizeStatements := normalizeStmtGen.generateNormalizeStatements(destinationTableName)
 		for _, normalizeStatement := range normalizeStatements {
```

Comment:
How many statements do we expect to generate? should we perhaps batch the normalize statements but not the tables?